odxtools/cli/browse.py

```diff
@@ -5,7 +5,7 @@
 from typing import List, Optional, Union, cast
 
 import InquirerPy.prompt as IP_prompt
-from tabulate import tabulate  # TODO: switch to rich tables
+# from tabulate import tabulate  # TODO: switch to rich tables
```

Comment:
commented out code

Reply:
I removed the tabulate table as the function call returns the rich table. So it can simply print it.

Reply:
okay